test/Tests/src/Semantic/Java_Interop_Spec.enso

```diff
@@ -53,4 +61,19 @@ spec =
             april1st.month.should_equal 4
             april1st.day.should_equal 1
 
+        Test.specify "Difference of Java Date and Enso date should be an Enso Duration" <|
```

Comment:
I think you might want to add those to Duration_Spec, similarly as we have date-specific tests in `Date_Spec`, `Date_Time_Spec` etc.